src/main/java/command/Command.java

```diff
@@ -0,0 +1,45 @@
+package command;
+
+import tasklist.TaskList;
+import utility.Storage;
+import utility.Ui;
+
+/**
+ * Represents all different types of command
+ */
+public abstract class Command {
+    private String input;
+
+    /**
+     * Instantiates a new command
+     */
```

Comment:
I think this is not very descriptive. Also, I think you forgot to include the parameters in this JavaDoc.